.github/workflows/scanner-release-vuln-update.yaml

```diff
@@ -77,16 +89,27 @@ jobs:
         go clean -cache -modcache
         ./bin/updater -output-dir=${{ env.ROX_PRODUCT_VERSION }}
```

Comment:
Should this be         `./bin/updater -output-dir=${{ env.ROX_PRODUCT_TAG }}`
And `gsutil cp -r "${{ env.ROX_PRODUCT_TAG }}" "gs://definitions.stackrox.io/v4/vulnerability-bundles"`
So we can get 4.4.0 Vulns bundle as expected